docs/serving/installing-logging-metrics-traces.md

```diff
@@ -5,6 +5,8 @@ weight: 40
 type: "docs"
 ---
 
+**DEPRECATED:** The monitoring tools released by Knative community are deprecated. Knative community will stop releasing them in a coming release.
```

Comment:
Maybe this should use the feature status short code?

Reply:
Yes, sounds good. Updated.